html/themes/custom/common_design_subtheme/components/gho-text/gho-text.css

```
@@ -1,3 +1,8 @@
 .paragraph--type--text {
-  max-width: 720px; /* reading-width */
+  max-width: var(--reading-width);
+}
+.paragraph--type--text p.highlight {
```

I randomly tried to add the Highlight style to the top Paragraph of the Section Overview, and CKEditor shows it but the public website doesn't because its selector was:

```

```

I think we can change to this and catch all of them:

```suggestion
.field--name-field-text .highlight {
```

Using ☝️ also works to make it slightly more wysiwyg in the editing screen as well.

---

Admin styles added to #98  (I'm PR bingo winner! GHO-98 on PR 98!)

---

@rupl, good catch, we should probably move that out of `gho-text` then because I'm not sure it's always loaded.

---

@rupl That said, thinking about that again, the style not applying outside of `Text` paragraph was on purpose. Other places share the sane ckeditor text format but according to the PDF prototype, this style is only to be used in `text` paragraphs.

---

ah shoot. Unfortunately I came back to this tab after lunch and proceeded with the changes but GitHub did not reflect your newest comments. I will make a quick follow-up and revert my commit.